src/RateHub/NewRelic/Providers/NewRelicServiceProvider.php

```diff
@@ -259,7 +259,7 @@ private function getJobName($event)
             ],
             [
                 $event->connectionName,
-                get_class($event->job),
+                $event->job->resolveName(),
```

Comment:
works in >= 5.6 https://github.com/laravel/framework/blob/5.6/src/Illuminate/Contracts/Queue/Job.php#L98

even in 5.1 didn't go further back since the package requires 5.6 at least